### ISSUE TITLE
Accessibility support in ngx-swiper for active carousel item/slide

### ISSUE THREAD

**Issue by pavandixit93** [zefoy/ngx-swiper-wrapper#225]
I am using ngx-swiper module in my project. For accessibility testing i am using NVDA/JAWS. When i hover on next/previous buttons it reads the text. But it does not read the content of the slides i.e current carousel item content.

It should work like, http://accessibility.athena-ict.com/aria/examples/carousel.shtml
On clicking next/previous button it reads the conent also.

Is there any way similar like above example so that after passing configuration for each carousel item, screen reader will read the actual content?

**Comment by sconix** [comment]
You can provide the buttons yourself for the Swiper, then you can make them behave as you wish.

**Comment by pavandixit93** [comment]
@sconix Thanks for the suggestion. I will do the same. 
Just for my information, This feature might be required for other users also, so are you going to add this feature itself in this library or is it in next roadmap?

**Comment by sconix** [comment]
Won't be part of this library since this is meant to be just a simple Angular wrapper for Swiper. The component version does add some convenient features over the directive, but no plans on expanding it.

**Comment by pavandixit93** [comment]
Closing as no action further.
